src/Raven.Studio/wwwroot/App/views/resources/databases.html

```diff
@@ -280,16 +283,22 @@ <h3 class="database-name">
     </div>
     <div class="properties-item item-last-backup" data-bind="visible: $root.createIsLocalDatabaseObservable(name)()">
         <div data-bind="attr: { class: 'properties-value value-only ' + backupStatus() }">
-            <div class="set-size" title="Last backup time"
-                 data-bind="css: { 'link-disabled': !canNavigateToDatabase() || isBeingDeleted() }">
+            <a class="set-size text-danger" title="Navigate to Ongoing Tasks view to create a backup"
```

Comment:
Shouldn't we redirect to new backups view ?

Reply:
oh it is for 5.0 so maybe we should update this on 5.1?

Reply:
yes - it should be updated for 5.1

Reply:
please create separate issue for that, ok ? we update that after this will be merged into 5.1.

Reply:
https://issues.hibernatingrhinos.com/issue/RavenDB-15896